src/cli/cli_coap.cpp

```diff
@@ -585,17 +576,21 @@ otError Coap::ProcessRequest(uint8_t aArgsLength, Arg aArgs[], otCoapCode aCoapC
     return error;
 }
 
-otError Coap::Process(uint8_t aArgsLength, Arg aArgs[])
+otError Coap::Process(Arg aArgs[])
 {
     otError        error = OT_ERROR_INVALID_ARGS;
     const Command *command;
 
-    VerifyOrExit(aArgsLength != 0, IgnoreError(ProcessHelp(0, nullptr)));
+    if (aArgs[0].IsEmpty())
+    {
+        IgnoreError(ProcessHelp(aArgs));
+        ExitNow();
+    }
```

Comment:
The main change is to pass `aArgs` instead of `nullptr` to `ProcessHelp()` (though it is ignored).

I also change the style to `if()` to make it similar/consistent with all other `Process{Cmd}()` methods (in CLI).

In general, I personally prefer to use `VerifyOrExit()` when the action is empty or just setting the `error`. For cases where we do some other action, I feel the `if(check) { action; ExitNow(); }` reads better. It's just matter of style and all are fine at the end.

Reply:
SGTM.